website/integrations: Omada Controller: cleanup (#23105)
goauthentik/authentik: website/integrations/networking/omada-controller/index.mdx

- `authentik.company` is the FQDN of the authentik installation.

> This guide applies to Omada Standard Cloud-Based Controller, Omada Software Controller, and OC300/OC400 hardware controllers that support SAML SSO.

Reviewer: Interesting, that OC200 does not support SSO according to https://support.omadanetworks.com/us/document/26917/

:::info
This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
:::

## authentik configuration

> To support the integration of Omada Controller with authentik, you need to create SAML property mappings, an application/provider pair, and application entitlements for the Omada SAML user groups users should receive.

Reviewer 2: ```suggestion
To support the integration of Omada Controller with authentik, you need to create SAML property mappings, an application/provider pair, and application entitlements to map users to Omada groups.
```

Not 100% on my recommendation but I think we need to improve the wording from the original.
